packages/ffe-form-react/src/InputGroup.js

```diff
@@ -9,7 +9,15 @@ import ErrorFieldMessage from './ErrorFieldMessage';
 class InputGroup extends Component {
     constructor(props) {
         super();
-        this.id = props.inputId ? props.inputId : `input-${uuid()}`;
+        const id = props.inputId ? props.inputId : `input-${uuid()}`;
+        this.id = id;
+        this.fieldMessageId = props.fieldMessage
+            ? (props.fieldMessage.props && props.fieldMessage.props.id) ||
+              `${id}-fieldmessage`
+            : undefined;
```

Comment:
Når du gjør dette i constructoren vil det vel bare finnes en `fieldMessageId` i de tilfeller der man sender inn en `fieldMessage` første gang man rendrer komponenten. Dette ser ut som det hadde passet bedre i en `useEffect` hook eller lignende, så kan komponenten skrives om til en functional component i samme slengen. 

Eventuelt lage en `fieldMessageId` selvom `props.fieldMessage` ikke er satt ved mount.

Reply:
Veldig godt poeng, jeg fikser!

Reply:
Muligens litt mer styr å lage test for, siden man må forholde seg til en lifecycle med endring av props. Jeg husker ikke lenger hvordan det er med enzyme, er en stund siden jeg har brukt det nå.

Reply:
Da var det fikset, @ivarni.

Reply:
Nice!